contents/tutorials/how-to-run-successful-group-experiments.md

```diff
@@ -0,0 +1,86 @@
+---
+title: How to run successful group-level A/B tests
+sidebar: Docs
+showTitle: true
+author: ['lior-neu-ner']
+date: // what to put here?
+featuredImage: //TODO
+tags: ['experimentation']
+---
+
+A/B tests are a powerful tool for measuring how product changes impact user behavior. However, sometimes changing how one person uses your product will affect how others interact with it too. For example, a change in the Uber App for Drivers will likely affect the experience of riders too. In these scenarios, group-level experiments allow us to measure the impact of changes beyond an individual and across a group of users.
+
+// TODO: Add image showing difference between group-level and user-level experiment
+
+## When to run group-level experiments
+
+When the change in how an individual uses your product will greatly affect the behavior of other users, or when you want to measure the impact of a change on an entire group of users. This is usually seen in the following types of products:
+
+1. B2B SaaS apps
+
+   - For example, suppose Asana wants to test a new AI feature that automatically assigns tasks based on their urgency and importance. By conducting a group-level experiment, they can measure the impact on project completion rates across the entire company. 
```

Comment:
Remove indents

Reply:
the indents are intentional. I'd like the bullets to be nested within the numbered list (I'll check that it's actually doing this on the site)

Reply:
Yup, ran it locally and works.